{"name": "Создание заказа с неверными хешами ингредиентов", "status": "failed", "statusDetails": {"message": "AssertionError: assert 500 == 400\n +  where 500 = <Response [500]>.status_code", "trace": "self = <test_create_order.TestCreateOrder object at 0x107f09820>\nauth_header = {'Authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY3NDdjZWRjOWVkMjgwMDAxYjUyNjdlNCIsImlhdCI6MTczMzI0NTc5NiwiZXhwIjoxNzMzMjQ2OTk2fQ.BLglDcOhMnHmpaU3sj2CtE0DtQEgI8BN91AtL9OZxJA'}\n\n    @allure.title(\"Создание заказа с неверными хешами ингредиентов\")\n    def test_create_order_with_invalid_ingredients(self, auth_header):\n        response = requests.post(\n            f\"{BASE_URL}{ENDPOINTS['orders']}\",\n            json={\"ingredients\": ORDER_DATA[\"invalid_ingredients\"]},\n            headers=auth_header\n        )\n>       assert response.status_code == 400\nE       assert 500 == 400\nE        +  where 500 = <Response [500]>.status_code\n\ntest_create_order.py:56: AssertionError"}, "start": 1733245797043, "stop": 1733245797216, "uuid": "9eeb089b-814f-4428-8e1e-a501059af066", "historyId": "8da158c03f34cca19354eabf273e4086", "testCaseId": "8da158c03f34cca19354eabf273e4086", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_invalid_ingredients", "labels": [{"name": "feature", "value": "Создание заказа"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "97049-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}